src/Groups/action.jl

```diff
@@ -0,0 +1,237 @@
+#############################################################################
+##
+##  common actions of group elements
+##
+##  The idea is to delegate the action of `GAPGroupElem` objects
+##  on `GAP.GapObj` objects to the corresponding GAP action,
+##  and to implement the action on native Julia objects case by case.
+
+export on_points, on_right, on_tuples, on_sets, permuted
+
+
+"""
+    on_points(pnt::GAP.GapObj, x::GAPGroupElem)
+    on_points(pnt::GAPGroupElem, x::GAPGroupElem)
+    on_points(pnt::Int, x::PermGroupElem)
```

Comment:
Honest question: What is the purpose in having `on_points` over just using `:^`? I.e. where one passes `on_points` as argument, couldn't one just pass `^` instead? It's quite likely I am missing something, but what? Note that I occasionally wonder the same about GAP.